docs/samples/llmisvc/dp-ep/qwen3-nvidia/llm-inference-service-dp-ep-multi-deepseek-r1-gpu-nvshmem-deepep-private.yaml

```diff
@@ -0,0 +1,271 @@
+            value: WARN
+          - name: NCCL_SOCKET_IFNAME
+            value: net1
+          - name: NCCL_IB_TIMEOUT
+            value: "100"
+          # NVSHMEM configuration - optimized for stability
+          - name: NVSHMEM_REMOTE_TRANSPORT
+            value: ibgda
+          - name: NVSHMEM_BOOTSTRAP_TWO_STAGE
+            value: "1"
+          - name: NVSHMEM_BOOTSTRAP_TIMEOUT
+            value: "300"
+          - name: NVSHMEM_BOOTSTRAP_UID_SOCK_IFNAME
+            value: net1
+          - name: NVSHMEM_IB_GID_INDEX
+            value: "3"
+          - name: NVSHMEM_USE_IBGDA
+            value: "1"
+          - name: NVSHMEM_ENABLE_NIC_PE_MAPPING
+            value: "1"
+          - name: NVSHMEM_IBGDA_SUPPORT
+            value: "1"
+          - name: NVSHMEM_IB_ENABLE_IBGDA
+            value: "1"
+          - name: NVSHMEM_IBGDA_NIC_HANDLER
+            value: "gpu"
+          # - name: NVSHMEM_IBGDA_RC_MAP_BY
+          #   value: "warp"
+          # - name: NVSHMEM_IBGDA_DCI_MAP_BY
+          #   value: "cta"
+          # - name: NVSHMEM_IBGDA_NUM_REQUESTS_IN_BATCH
+          #   value: "32"
+          - name: NVSHMEM_DEBUG
+            value: WARN
+          # UCX configuration for NVSHMEM
+          - name: UCX_TLS
+            value: "rc,sm,self,cuda_copy,cuda_ipc"
+          - name: UCX_IB_GID_INDEX
+            value: "3"
+          - name: UCX_RC_MLX5_TM_ENABLE
+            value: "n"
+          - name: UCX_UD_MLX5_RX_QUEUE_LEN
+            value: "1024"
+          - name: NVIDIA_GDRCOPY
+            value: enabled
+        resources:
+          limits:
+            cpu: 128
+            ephemeral-storage: 800Gi
+            memory: 512Gi
+            nvidia.com/gpu: "8"
+            rdma/roce_gdr: 1
+          requests:
+            cpu: 64
+            ephemeral-storage: 800Gi
+            memory: 256Gi
+            nvidia.com/gpu: "8"
+            rdma/roce_gdr: 1
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 8000
+            scheme: HTTPS
+          initialDelaySeconds: 4800
+          periodSeconds: 10
+          timeoutSeconds: 10
+          failureThreshold: 3
+        securityContext:
+          runAsNonRoot: true
+          allowPrivilegeEscalation: false
+          capabilities:
+            add:
+              - "IPC_LOCK"
+              - "SYS_RAWIO"
+              - "NET_RAW"
+            drop:
+              - ALL
+          seccompProfile:
+            type: RuntimeDefault
+  worker:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/hostname
+                  operator: In
+                  values:
+                    - pokprod-b93r38s0
+                    - pokprod-b93r38s1
+                    - pokprod-b93r39s1
+                    - pokprod-b93r38s2
+                    - pokprod-b93r38s3
```

Comment:
These look very specific - from docs perspective maybe worth emphasizing it's an example list of hostnames for node selection?